packages/manager/src/features/Billing/BillingPanels/ContactInfoPanel/ContactInformation.tsx

```diff
@@ -85,7 +85,7 @@ const ContactInformation = (props: Props) => {
   const isChildUser = Boolean(profile?.user_type === 'child');
 
   const invalidTaxIdNotification = notifications?.find((notification) => {
```

Comment:
We should probably rename this const to be consistent

```suggestion
  const taxIdIsVerifyingNotification = notifications?.find((notification) => {
```